src/main/java/com/palantir/ri/ResourceIdentifier.java

```diff
@@ -104,43 +113,39 @@ public String getType() {
      * @return the locator component from this identifier
      */
     public String getLocator() {
-        return locator;
+        CharBuffer.wrap(resourceIdentifier);
```

Comment:
?

Reply:
Left over from some perf testing 😞

I’ll fix when I’m back in the office.